src/acquisition/covidcast/database.py

```diff
@@ -213,10 +213,15 @@ def insert_or_update_batch(self, cc_rows, batch_size=2**20, commit_partial=False
 
 
         result = self._cursor.executemany(insert_into_tmp_sql, args)
+
+        meta_update = self.compute_covidcast_meta(table_name=tmp_table_name)
```

Comment:
annotating since i definitely missed this the first time through

```suggestion
        # use the temp table to compute meta just for new data, 
        # then merge with the global meta
        meta_update = self.compute_covidcast_meta(table_name=tmp_table_name)
```